src/main/java/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -1921,6 +1927,14 @@ private void parseBind(Element element) {
     private void addBinding(DataBinding binding) {
         bindings.add(binding);
 
+        IDataReference reference = binding.getReference();
+        Optional<TreeElement> maybeSaveTo = binding.getAdditionalAttributes().stream().filter(treeElement -> treeElement.getName().equals("saveto")).findFirst();
```

Comment:
Should eventually be a known attribute rather than going to `getAdditionalAttributes`

Reply:
"saveto" should not remain in `getAdditionalAttributes()` (clients shouldn't be able to access it). You should be able to call `binding.getSaveTo()`.

Reply:
Or, maybe `saveto` should live somewhere else entirely and not in `getAdditionalAttributes()` or on the `DataBinding`.